apps/chat_oauth2/lib/chat_oauth2/current_user.ex

```diff
@@ -0,0 +1,11 @@
+defmodule ChatOAuth2.CurrentUser do
+  def present?(%{context: %{current_user: %{id: _id}}}), do: true
+  def present?(_), do: false
+
+  def id(%{context: %{current_user: %{id: id}}}), do: id
+  def id(_, _), do: nil
```

Comment:
Why does this function output `nil` here? Is there another place where the case of a `nil` id raises an error?